app-android-xamarin/Moodis/Moodis/Feature/Register/RegisterViewModel.cs

```diff
@@ -52,8 +55,14 @@ public async Task<Response> DeleteUser()
             return await Face.Instance.DeletePerson(currentUser.PersonGroupId);
         }
 
+        //TODO MOVE THIS TO FACE CLASS
         public async Task<Response> AddFaceToPerson(string imagePath)
         {
+            if (currentUser == null)
+            {
+                currentUser = SignInViewModel.currentUser;
```

Comment:
why do you need this assignment?

Reply:
it tries to take current user from register view model in which currentUser is not defined because the person is not registering.

Reply:
But we use `RegisterViewModel's` ``currentUser` in only few places? If so, then it might be better to call `SignInViewModel's` `currentUser`? I mean, it is much better to have only one source of truth for this property

Reply:
Yeah we SHOULD have current user in only one place however as it is now it's in singinViewModel and RegisterViewModel which both get current user on diferent instances (registering/signin in).  Don't know why aurimas created a new one for registration purposes.